src/lib/routes/admin-api/environments.ts

```diff
@@ -58,8 +61,12 @@ export class EnvironmentsController extends Controller {
             middleware: [
                 openApiService.validPath({
                     tags: ['Environments'],
+                    description: 'Gets all environments',
```

Comment:
This should probably be a summary and not a description.

We haven't discussed this previously and it's not written down (I'll do it 🙋🏼), but summaries should probably be expressed as a "bare infinitive", as in "call this endpoint to [summary]".



```suggestion
                    summary: 'Get all environments',
                    description: 'Retrieves all environments that exist in this Unleash instance.'
```